lib/spack/spack/compilers/aocc.py

```diff
@@ -98,15 +98,12 @@ def fc_pic_flag(self):
     @classmethod
     @llnl.util.lang.memoized
     def extract_version_from_output(cls, output):
-        loc_ver = 'unknown'
-
         match = re.search(
-            r'AMD clang version ([^ )]+)',
+            r'AOCC_(\d)[._](\d)[._](\d)',
```

Comment:
Now that we are using `AMD clang version ...` instead of `AOCC ...` are the underscores actually necessary?